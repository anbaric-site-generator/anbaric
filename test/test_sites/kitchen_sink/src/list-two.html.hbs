---
foo: bar
---
<h1>Top-level</h1>
<h2>{{ foo }}</h2>
<ul>
    {{#each site.pages}}
        <li>{{ url }}</li>
    {{/each}}
</ul>