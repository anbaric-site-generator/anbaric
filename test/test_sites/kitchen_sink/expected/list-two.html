<html>
  <head></head>
  <body>
    <h1>This is the layout</h1>
    <h1>Top-level</h1>
    <h2>bar</h2>
    <ul>
      <li>/index.html</li>
      <li>/list-two.html</li>
      <li>/page2.html</li>
    </ul>
  </body>
</html>